### PR TITLE
Add openSUSE 15.3 package builds.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

SSIA

##### Component Name

area/ci

##### Test Plan

CI checks pass.

### CODE REVIEW

**Review by kaskavel — DISMISSED** [review]

File: packaging/scripts/install.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ install_suse_like() {
 
   # Install testing tools
   zypper install -y --no-recommends \
-    curl gnu-netcat jq
+    curl netcat-openbsd jq
```
Comment (vkalintiris) [inline packaging/scripts/install.sh:63]: @Ferroin Out of curiosity, why is this change required?

**Review by vkalintiris — APPROVED** [review]

File: packaging/scripts/install.sh (hunk 1 above)
Comment (Ferroin) [reply packaging/scripts/install.sh:63]: OpenSUSE has apparently stopped shipping the GNU version of netcat for some reason. We end up using netcat when checking the packages as the last part of the CI checks (and as a final layer of insurance before uploading them during nightly builds and release builds), so we need a working copy of netcat.
